pulp_puppet_plugins/pulp_puppet/plugins/importers/directory.py

```diff
@@ -262,8 +262,8 @@ def _import_modules(self, module_paths):
             _logger.debug(IMPORT_MODULE, dict(mod=module_path))
 
             module.set_storage_path(os.path.basename(module_path))
```

Comment:
After talking about this with @jeremycline I realize that the `set_content_path()` call and the `import_content()` call must both occur and in that order. I didn't realize this was required with the new import_content() interface. Given that is how platform works, we should merge this and re-evaluate that as a separate issue.